lms/djangoapps/courseware/views.py

```diff
@@ -737,7 +737,7 @@ def course_info(request, course_id):
         return render_to_response('courseware/info.html', context)
 
 
-def get_last_accessed_courseware(course, request):
+def get_last_accessed_courseware(course, request, user):
     """
     Return the URL the courseware module that this request's user last
     accessed, or None if it cannot be found.
```

Comment:
I think this should be changed to => `Return the courseware module URL that the user last accessed`.
as that we are not using `requests's user`